lib/views/item-scene.js

```diff
@@ -18,21 +21,34 @@ const {
   Linking,
 } = ReactNative;
 
+const accessToken = 'pk.eyJ1IjoiZ21hcnR5IiwiYSI6ImNpdm1lanB1azAwNm8yb3A0OXZ6bmUxM2oifQ.i2HycDMBdPoatAB-ONR6FQ';
+Mapbox.setAccessToken(accessToken);
```

Comment:
What are the rate limits for mapbox? We may need to flip to google-maps if we start entering paid usage tiers.